doc/devel/coding_style_guideline.rst

```diff
@@ -35,6 +35,25 @@ compliance of your code with PEP8. Beyond the aspects checked, as a
 contributor to DIPY, you should try to ensure that your code, including
 comments, conform to the above principles.
 
+-------------------
+Cython coding style
+-------------------
+DIPY recommends the use of the standard Python
+`PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ style when writing
+`Cython <https://cython.org/>` code. The sole exception to these guidelines is
+the line length limitation recommendation: a line length of 120 characters is
```

Comment:
Is there any reason for this?   Of course some people like longer lines, but there's good reason for the shorter line length in PEP8.

Reply:
Commented above.